src/main/java/org/researchspace/sail/rest/RESTSailConnection.java

```diff
@@ -141,8 +149,10 @@ public RESTSailConnection(AbstractServiceWrappingSail<RESTSailConfig> sailBase)
 
             // Evaluate each jsonPath singularly and add the result to the binding
             for (Map.Entry<String, String> path : jsonPaths.entrySet()) {
-                mapBindingSet.addBinding(path.getKey(),
-                        VF.createLiteral(JsonPath.read(object, path.getValue()).toString(), XSD.STRING));
+                Object value = JsonPath.using(this.jsonPathConfig).parse(object).read(path.getValue());
+                if (value != null) {
+                    mapBindingSet.addBinding(path.getKey(), VF.createLiteral(value.toString(), XSD.STRING));
+                }
```

Comment:
I have addressed the same issue in https://github.com/researchspace/researchspace/pull/211/commits/1ce826f162247c230a430a9cae0e968f3056bd74. It will need a manual review when merging.